codec/test_codec.go

```diff
@@ -684,13 +684,13 @@ func TestSliceWithEmptySerialization(codec GeneralCodec, t testing.TB) {
 
 	bytesLen, err := manager.Size(0, val)
 	require.NoError(err)
-	require.Equal(len(result), bytesLen)
+	require.Len(result, bytesLen)
 
 	unmarshaled := nestedSliceStruct{}
 	version, err := manager.Unmarshal(expected, &unmarshaled)
 	require.NoError(err)
 	require.Zero(version)
-	require.Equal(1000, len(unmarshaled.Arr))
```

Comment:
ex: this is a good change, but the above changes I don't think result in a good error message